incubator/elasticsearch/values-production.yaml

```diff
@@ -0,0 +1,211 @@
+## Bitnami Elasticsearch image version
+## ref: https://hub.docker.com/r/bitnami/elasticsearch/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/elasticsearch
+  tag: 6.2.3-r3
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+## Elasticsearch cluster name
+##
+name: elastic
+
+## The serviceAccountName to be used, required by the elasticsearch-cloud-kubernetes plugin
+## ref: https://github.com/fabric8io/elasticsearch-cloud-kubernetes#kubernetes-auth
+##
+serviceAccountName: elastic
+
+## Comma seperated list of plugins
+## ref: https://github.com/bitnami/bitnami-docker-elasticsearch#environment-variables
+##
+plugins: http://github.com/fabric8io/elasticsearch-cloud-kubernetes/archive/elasticsearch-cloud-kubernetes-6.2.3.zip
+
+## Customize elasticsearch configuration
+## ref: https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html
+##
+config:
+
+## Elasticsearch master-eligible node parameters
+##
+master:
+  name: master
+  replicas: 3
+  heapSize: 128m
+  antiAffinity: "soft"
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
```

Comment:
is not recommended to set default values for resources. Do you have a strong reason to do it?

Reply:
You are right @tompizmor. Having worked on this chart before, the elasticsearch nodes have very hefty resource requirements (mostly RAM). It makes sense to define the resource requests in this case so that the nodes are scheduled properly.

Reply:
All right! We'll keep only the "data" ones. What about the **heapSize** @sameersbn ? It's closely related with the resources.

Reply:
yea.. mainly because we defined heapSize we want to keep the resources. I would not make any changes there

Reply:
👍